manual/fr/shadows.html

```diff
@@ -161,7 +161,7 @@ <h1>Les ombres dans </h1>
     const u = ndx / sphereShadowBases.length;
 
     // calculer une position pour la base. Cela va bouger
-    // à la fois la sphère et son ombre
+    // à la fois la sphère et child ombre
```

Comment:
I don't think this correction is right. The text here is French, not English.

Reply:
... unless "Child hombre" is come Spanish slang and it's missing only an `h`